news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,208 @@
+
+<iframe width="100%" height="315" src="https://youtu.be/embed/hKwIKC-DvcU" frameborder="0" allowfullscreen></iframe>
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+
+#### Superstars Strike
+
+The mania k-pop community is going to be absolutely ecstatic at this next showcase. 
+
+With [Onta_Bekasi](https://osu.ppy.sh/users/8677684) not just clearing, but FCing the deceptively difficult yet soothing myriad of LNs and intricate patterns that [My Man](https://osu.ppy.sh/beatmapsets/514681#mania/1093212) so elegantly puts on display, it is of incredible skill, and no small effort, that [Onta_Bekasi](https://osu.ppy.sh/users/8677684) only suffers a 00.02 accuracy drop across a 3-minute map and over 5000 notes. Truly a wonderful display of what osu!mania can offer, enjoyment of the game, and is a play that is more than deserving of being showcased.
+
+<iframe width="100%" height="315" src="https://youtu.be/embed/1RuyJ3y_ot4" frameborder="0" allowfullscreen></iframe>
+
+#### Millions of Long Notes
+
+Wow. Just completely shocked at this performance by [[GS]Myuka](https://osu.ppy.sh/users/10072733) on [Camellia’s - MI11I0N PP](https://osu.ppy.sh/beatmapsets/1388481#mania/2867552). 
+
+If you thought *My Man* had LNs, wait until you see a map of over 7 minutes of non-stop LN gameplay, and with not just single-note rainbows and glissandos, but an interconnected web of the two. It is an impeccable display of skill, accuracy, speed, and just what osu!monthly is looking for in a showcase!
+
+<iframe width="100%" height="315" src="https://youtu.be/embed/nc_ejGtZKUM" frameborder="0" allowfullscreen></iframe>
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+#### Galaxy Brain
+
+Somebody scream! Even with being the number one osu!catch player in the world, a play such as [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) is no small feat. 
+
+Coming in with a HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock! 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>
+
+---
+
+## New Features on osu!(lazer)!
+
+The new client has never so **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 
+For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
```

Comment:
```suggestion

For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
```
Should be a separate paragraph in my opinion (markdown will otherwise show it as one).